bugsnag-android-core/src/main/java/com/bugsnag/android/internal/JournalKeys.kt

```diff
@@ -0,0 +1,59 @@
+package com.bugsnag.android.internal
+
+object JournalKeys {
```

Comment:
The problem is that you have the same key values in different objects, and this will happen more as we add more information to the document.

Reply:
We could handle the duplicate-key issue by having the keys at multiple layers:

```object JournalKeys {
   internal const val totalMemory = "totalMemory"
   
   object Device {
      internal const val totalMemory = JournalKeys.totalMemory
   }
   
   object App {
      internal const val totalMemory = JournalKeys.totalMemory
   }
```

It might be a bit verbose, but the compiler will pack all of these down since they are all `const`.

Reply:
The latest update adds `toJournalSection` methods to various data objects. I think this clears things up a fair bit since the keys are used in the objects themselves now.